internal/pkg/cli/deploy/lbws.go

```diff
@@ -188,13 +188,27 @@ func (d *lbWebSvcDeployer) stackConfiguration(in *StackRuntimeConfiguration) (*s
 }
 
 func (d *lbWebSvcDeployer) validateALBRuntime() error {
```

Comment:
Is there any way we can reduce the code duplication in this function? It seems to appear in LBWS and Backend service and I wonder if there's an opportunity to extract this to a function that can be called from both BackendService.validateALBRuntime and LBWS.validateALBRuntime. Maybe there are too many variables though.

Reply:
Yeah, the problem with making these two functions into one is the number of parameters we need to send and the different runtime requirements that these two services hold.